lisa/node.py

```diff
@@ -514,3 +514,24 @@ def from_requirement(
         self._list.append(node)
 
         return node
+
+
+def quick_connect(runbook: schema.Node, name: str = "", index: int = -1) -> Node:
```

Comment:
How about we start documenting arguments and return values as well? Here, people might think name is for node name (which is not a bad idea), but it's not the case

Reply:
The "name" is not a good name, update it to “logger_name”.